### PR TITLE
Test0

### PR DESCRIPTION
**Pull request by esirK**
# Status
In Development
## Description
This tests for `create_room <room_type> <room_name>` and `add_person <person_name> <FELLOW|STAFF> [wants_accommodation]` functionality as Specified in Test0 at [Andela](https://docs.google.com/document/d/1ya03jwGaEn5JJBb3sKu4FEQ74tzIb8cVIMlZV23-bzU/edit#)

### Todos
Please Review The Following Files
Classes/Person/Person/test_person.py
Classes/Room/rooms/test_rooms.py

### CODE REVIEW

File: Classes/rooms/rooms.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    def create_room(self,name,what_type):
+        self.what_type=what_type
+        asmany_names=[]
+        asmany_names.append(name)
+        if self.what_type=="LIVING_ROOM":
+            create_living_room=LivingSpace(name)
+            create_living_room
+        else:
+            creat_office=Office(name)
+            creat_office
```
Comment (george-waf) [inline Classes/rooms/rooms.py]: create_office

File: Classes/Person/test_person.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    def test_person_type_staff(self):
+        staff=self.x_staff.add_person("Mas","Staff")
+        #print (staff[0])
+        self.assertEqual(staff[0],"Staff")
+    
+    def test_person_type_fellow(self):
+        fellow=self.x_fellow.add_person("Willi","Fellow","Y")
+        #print (fellow[0])
+        self.assertEqual(fellow[0],"Fellow")
+
```
Comment (george-waf) [inline Classes/Person/test_person.py]: `gffg`

File: Classes/rooms/rooms.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
```
Comment (george-waf) [inline Classes/rooms/rooms.py]: `create_office=Office(name)`
`create_office`

Comment (esirK) [reply Classes/rooms/rooms.py]: Yeah I Should remove the repetition

File: Classes/Person/person.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+    persons=[]
+    rooms=[]
+    def __init__(self,rooms):
+        self.rooms=rooms
+        #print (self.rooms)
+
+    def add_person(self,name,p_type,accommodation="N"):
+        Person.num_of_persons_at_dojo+=1
+        self.name=name
+        self.p_type=p_type
```
Comment (ghost) [inline Classes/Person/person.py]: Use more descriptive variables.

File: Classes/rooms/rooms.py [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    rooms=[]
+    def __init__(self,name,what_type):
+        self.all_rooms+=1
+        self.what_type=what_type
+
+    def __init__(self):
+        pass
+
+    def create_room(self,name,what_type):
+        self.what_type=what_type
```
Comment (victoriaaoka) [inline Classes/rooms/rooms.py]: Make the variable names more descriptive.
Avoid using variable names such as 'what_type', 'as_many'

File: Classes/Person/person.py [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+        pers.append(p_type)
+
+        #Add a person to room Randomly
+        if self.p_type=="Staff":
+            self.accommodation="N"
+            add_a_staffx=Staff().create_staff(name,self.rooms)
+            #Add Staff To Database/TextFile
+            print (add_a_staffx)
+            #return Room Allocated
+
```
Comment (george-waf) [inline Classes/Person/person.py]: should it be `add_a_staffx` or `add_a_staff`

File: Classes/Person/test_person.py [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    def test_person_type_staff(self):
+        staff=self.x_staff.add_person("Mas","Staff")
+        #print (staff[0])
+        self.assertEqual(staff[0],"Staff")
+    
+    def test_person_type_fellow(self):
+        fellow=self.x_fellow.add_person("Willi","Fellow","Y")
+        #print (fellow[0])
+        self.assertEqual(fellow[0],"Fellow")
```
Comment (victoriaaoka) [inline Classes/Person/test_person.py]: Nice tests. consider using different assertions

Comment (esirK) [reply Classes/Person/test_person.py]: I will Add Some More and Commit Them. Thanks For that

File: Classes/Person/person.py (hunk 6 above)
Comment (esirK) [reply Classes/Person/person.py]: should be `add_a_staffx` I admit my Variable names are poor. I'll rename them to be more descriptive

**Review by bernard-kanyolo — APPROVED** [review]
Key points:-
- do not commit *.pyc files
- your code should be pep8 compliant
- your commit messages should follow this standard https://chris.beams.io/posts/git-commit/

File: Classes/Dojo/Dojo_main.py [hunk 8]
```diff
@@ -0,0 +1,223 @@
...
+from Person.person_new import Person
+from Person.Staff.staff import Staff
+from rooms.room_new import Room
+import random
+import collections
+
+
+class Dojo(object):
+    def __init__(self):
+        self.list_persons = []
```
Comment (mungaiandela) [inline Classes/Dojo/Dojo_main.py]: Use class variables for data structures. This structures will be initialized every time class Dojo runs.

**Pull request closed without merging**